content/questions/actions/question-117.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "Who can bypass configured deployment protection rules to force deployment (by default)"
+archetype: "questions"
+title: "Question 117"
+draft: false
+---
+
+> https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#allow-administrators-to-bypass-configured-protection-rules
+1. [x] Administrators
+1. [ ] Owners
```

Comment:
Why would this be incorrect? I'm the owner of this repository and I can bypass anything

Reply:
hm yep, it works, I have deleted this option